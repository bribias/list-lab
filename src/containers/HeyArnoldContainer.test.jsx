import React from 'react';
import { render, screen } from '@testing-library/react';
import HeyArnoldContainer from './HeyArnoldContainer';
import { MemoryRouter } from 'react-router-dom';

describe('HeyArnoldContainer', () => {
    it('renders a list of characters', async () => {
        render(<MemoryRouter><HeyArnoldContainer /></MemoryRouter>);

        screen.getAllByAltText('loading spinner');

        const ul = await screen.findByRole('list', { name: 'characters' });
        expect(ul).toMatchSnapshot();
    });
});