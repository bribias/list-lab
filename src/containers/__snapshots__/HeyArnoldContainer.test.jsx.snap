// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeyArnoldContainer renders a list of characters 1`] = `
<ul
  aria-label="characters"
  role="list"
>
  <li>
    <p>
      Arnold Shortman
    </p>
    <p>
      https://vignette.wikia.nocookie.net/heyarnold/images/f/f5/Arnold.png/revision/latest?cb=20181027162333
    </p>
  </li>
  <li>
    <p>
      Grandpa Phil
    </p>
    <p>
      https://vignette.wikia.nocookie.net/heyarnold/images/7/79/GrandpaPhil.png/revision/latest/scale-to-width-down/300?cb=20160924030436
    </p>
  </li>
  <li>
    <p>
      Grandma Gertie
    </p>
    <p>
      https://vignette.wikia.nocookie.net/heyarnold/images/7/7b/DB87AC2A-C2C2-4DAB-AECF-B9F7273C746B.png/revision/latest/scale-to-width-down/306?cb=20190816101257
    </p>
  </li>
  <li>
    <p>
      Mitzi
    </p>
    <p>
      https://vignette.wikia.nocookie.net/heyarnold/images/d/d9/Mitzi.png/revision/latest/scale-to-width-down/310?cb=20171212070331
    </p>
  </li>
  <li>
    <p>
      Arnie
    </p>
    <p>
      https://vignette.wikia.nocookie.net/heyarnold/images/4/42/Arnie.jpg/revision/latest/scale-to-width-down/310?cb=20110109195825
    </p>
  </li>
  <li>
    <p>
      Miles Shortman
    </p>
    <p>
      https://vignette.wikia.nocookie.net/heyarnold/images/2/2e/Miles.jpg/revision/latest/scale-to-width-down/243?cb=20101115140639
    </p>
  </li>
  <li>
    <p>
      Stella Shortman
    </p>
    <p>
      https://vignette.wikia.nocookie.net/heyarnold/images/f/fa/Stella.jpg/revision/latest?cb=20101225143956
    </p>
  </li>
  <li>
    <p>
      Grandpa's father
    </p>
    <p>
      https://vignette.wikia.nocookie.net/heyarnold/images/e/e0/Grandpa%27s_father.png/revision/latest/scale-to-width-down/310?cb=20190228081112
    </p>
  </li>
  <li>
    <p>
      Grandpa's grandfather
    </p>
    <p>
      https://vignette.wikia.nocookie.net/heyarnold/images/f/f7/Grandpa%27s_grandfather.png/revision/latest/scale-to-width-down/310?cb=20190228081015
    </p>
  </li>
  <li>
    <p>
      Oskar Kokoshka
    </p>
    <p>
      https://vignette.wikia.nocookie.net/heyarnold/images/c/c5/Oskar_Kokoshka.png/revision/latest/scale-to-width-down/310?cb=20180707032418
    </p>
  </li>
  <li>
    <p>
      Suzie Kokoshka
    </p>
    <p>
      https://vignette.wikia.nocookie.net/heyarnold/images/f/fe/Suzie.jpg/revision/latest/scale-to-width-down/310?cb=20110223003447
    </p>
  </li>
  <li>
    <p>
      Nancy
    </p>
    <p>
      https://vignette.wikia.nocookie.net/heyarnold/images/6/62/Nancy.jpg/revision/latest/scale-to-width-down/310?cb=20110423055923
    </p>
  </li>
  <li>
    <p>
      Ernie Potts
    </p>
    <p>
      https://vignette.wikia.nocookie.net/heyarnold/images/6/67/Ernie.jpeg/revision/latest/scale-to-width-down/310?cb=20181213181427
    </p>
  </li>
  <li>
    <p>
      Mr. Hyunh
    </p>
    <p>
      https://vignette.wikia.nocookie.net/heyarnold/images/6/60/Mr._Hyunh.png/revision/latest/scale-to-width-down/310?cb=20180707031441
    </p>
  </li>
  <li>
    <p>
      Mai Hyunh
    </p>
    <p>
      https://vignette.wikia.nocookie.net/heyarnold/images/8/83/Mai_Hyunh.jpeg/revision/latest/scale-to-width-down/310?cb=20190118192457
    </p>
  </li>
  <li>
    <p>
      Mr. Smith
    </p>
    <p>
      https://vignette.wikia.nocookie.net/heyarnold/images/8/80/Mr._Smith.jpg/revision/latest?cb=20110206233402
    </p>
  </li>
  <li>
    <p>
      Lana Vail
    </p>
    <p>
      https://vignette.wikia.nocookie.net/heyarnold/images/e/e8/762C69BE-23BA-48BA-9B14-6AF3246FCD3F.jpeg/revision/latest/scale-to-width-down/304?cb=20190106211607
    </p>
  </li>
  <li>
    <p>
      Mr. Purdy (character)
    </p>
    <p>
      https://vignette.wikia.nocookie.net/heyarnold/images/9/9c/Mr._Purdy_%28character%29.jpeg/revision/latest?cb=20190125214742
    </p>
  </li>
  <li>
    <p>
      Helga G. Pataki
    </p>
    <p>
      https://vignette.wikia.nocookie.net/heyarnold/images/e/e5/Helga_Pataki.png/revision/latest/scale-to-width-down/266?cb=20190816021807
    </p>
  </li>
  <li>
    <p>
      Robert "Big Bob" Pataki
    </p>
    <p>
      https://vignette.wikia.nocookie.net/heyarnold/images/8/89/Big_Bob.jpg/revision/latest/scale-to-width-down/310?cb=20170428021615
    </p>
  </li>
</ul>
`;
